chef/data_bags/crowbar/template-provisioner.json

```diff
@@ -9,6 +9,7 @@
       "access_keys": "",
       "shell_prompt": "USER@HOST:CWD SUFFIX",
       "enable_pxe": true,
+      "self_update_url": "http://crowbar/sle12/$arch",
```

Comment:
- I've add the `<ADMINWEB>` alias.
- Opened doc bug: https://bugzilla.suse.com/show_bug.cgi?id=1012469
- I reverted the default to "" which defaults to `scc.suse.com`. Or would you prefer a versioned local URL?